sdk/communication/Azure.Communication.Common/tests/CommunicationTestEnvironment.cs

```diff
@@ -9,17 +9,21 @@ namespace Azure.Communication.Tests
     public class CommunicationTestEnvironment : TestEnvironment
     {
         public const string ConnectionStringEnvironmentVariableName = "COMMUNICATION_CONNECTION_STRING";
+        public const string LiveTestConnectionStringEnvironmentVariableName = "AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING";
+        public const string AzurePhoneNumber = "AZURE_PHONE_NUMBER";
 
         public string ConnectionString => GetRecordedVariable(ConnectionStringEnvironmentVariableName);
 
         public Uri Endpoint => new Uri(Core.ConnectionString.Parse(ConnectionString).GetRequired("endpoint"));
 
         public string AccessKey => Core.ConnectionString.Parse(ConnectionString).GetRequired("accesskey");
 
-        internal const string LiveTestConnectionStringEnvironmentVariableName = "AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING";
-
         public string LiveTestConnectionString => GetRecordedVariable(LiveTestConnectionStringEnvironmentVariableName);
 
         public Uri LiveTestEndpoint => new Uri(Core.ConnectionString.Parse(LiveTestConnectionString).GetRequired("endpoint"));
+
+        public string LiveTestAccessKeyAccessKey => Core.ConnectionString.Parse(LiveTestConnectionString).GetRequired("accesskey");
+
+        public string CommunicationTestPhoneNumber => GetVariable(AzurePhoneNumber);
```

Comment:
let's move the PhoneNumber variables to `PhoneNumbersClientTestEnvironment` instead of the common env variables.

Reply:
I think the reason it's here is because SMS shares the same phone number

Reply:
let's leave it here for now then, but let's talk about making this better. I have some ideas